.github/workflows/ci.yaml

```diff
@@ -609,34 +529,20 @@ jobs:
           # 0 is required here for version.sh to work.
           fetch-depth: 0
 
-      - name: Setup Node
-        uses: ./.github/actions/setup-node
-        with:
-          directory: offlinedocs
-
-      - name: Setup Go
-        uses: ./.github/actions/setup-go
+      - name: Setup Nix
```

Comment:
Offline docs also need to run a Postgres container, so not running it using the container.